cwltool/process.py

```diff
@@ -482,23 +482,17 @@ def __init__(self, toolpath_object, **kwargs):
                 elif key == "outputs":
                     self.outputs_record_schema["fields"].append(c)
 
-        try:
-            self.inputs_record_schema = cast(Dict[six.text_type, Any], schema_salad.schema.make_valid_avro(self.inputs_record_schema, {}, set()))
+        with SourceLine(toolpath_object, "inputs",
+                avro.schema.SchemaParseException):
```

Comment:
This will cause it to raise a SchemaParseException instead of catching it and converting it into a ValidationException.  SchemaParseException is an unknown exception that will result in a backtrace by default whereas ValidationException won't.  Is that intentional?